pkg/streamingpromql/benchmarks/benchmarks.go

```diff
@@ -190,6 +204,9 @@ func TestCases(metricSizes []int) []BenchCase {
 		{
 			Expr: "sum by (le)(rate(h_X[1m]))",
 		},
+		{
+			Expr: "sum by (l)(rate(nh_X[1m]))",
```

Comment:
I'm not sure this is such an interesting benchmark - each `nh_...` series has a unique value for `l`, so each input series maps to its own output series (eg. `nh_100` has 100 series, each with a different value for `l`). I think anything this benchmark will tell us will be covered by other cases, so I think we can remove it.